test/benchmark/fuse_db/bin/run_sqlite.sh

```
@@ -0,0 +1,67 @@
+  [ "$p" -ge $((TOTAL/2)) ] && break
+  kill -0 $GENPID 2>/dev/null || { say "loader exited early"; break; }
+  sleep 0.2
+done
+LB=$(cat "$PROG" 2>/dev/null || echo 0)
+say "durable committed lower-bound at crash = $LB rows; crashing storage now"
+hard_kill_all                      # storage dies under the still-writing loader
+kill -9 $GENPID 2>/dev/null; wait $GENPID 2>/dev/null
+sleep 1
+cluster_start || exit 1; mount_start || exit 1
+say "verifying recovery (committed prefix must survive, integrity must hold)"
+python3 "$BIN/sqlite_verify.py" "$DB" $LB atleast; chk $? "C/crash-during-write"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)